panic: the originalTemplate must not be nil, please use OriginalTemplate() function instead

```
panic: the originalTemplate must not be nil, please use OriginalTemplate() function instead
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: 
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: goroutine 16 [running]:
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: github.com/imjoey/terraform-provider-ovirt/vendor/github.com/ovirt/go-ovirt.(*Vm).MustOriginalTemplate(...)
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/git/go/src/github.com/imjoey/terraform-provider-ovirt/vendor/github.com/ovirt/go-ovirt/types.go:22870
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: github.com/imjoey/terraform-provider-ovirt/ovirt.resourceOvirtVMRead(0xc0004e77a0, 0x17fde00, 0xc0001a0640, 0x1, 0x1)
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/git/go/src/github.com/imjoey/terraform-provider-ovirt/ovirt/resource_ovirt_vm.go:504 +0xb02
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: github.com/imjoey/terraform-provider-ovirt/ovirt.resourceOvirtVMCreate(0xc0004e77a0, 0x17fde00, 0xc0001a0640, 0xc0004e77a0, 0x0)
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/git/go/src/github.com/imjoey/terraform-provider-ovirt/ovirt/resource_ovirt_vm.go:420 +0x18c0
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: github.com/imjoey/terraform-provider-ovirt/vendor/github.com/hashicorp/terraform/helper/schema.(*Resource).Apply(0xc000156700, 0xc0004ea780, 0xc0003fad60, 0x17fde00, 0xc0001a0640, 0x40d501, 0xc000499b80, 0x4c3cac)
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/git/go/src/github.com/imjoey/terraform-provider-ovirt/vendor/github.com/hashicorp/terraform/helper/schema/resource.go:286 +0x363
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: github.com/imjoey/terraform-provider-ovirt/vendor/github.com/hashicorp/terraform/helper/schema.(*Provider).Apply(0xc000157e00, 0xc0004ea730, 0xc0004ea780, 0xc0003fad60, 0xc000073180, 0x18, 0x7f19e5b8b000)
2019-06-03T15:57:22.885-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/git/go/src/github.com/imjoey/terraform-provider-ovirt/vendor/github.com/hashicorp/terraform/helper/schema/provider.go:285 +0x9c
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: github.com/imjoey/terraform-provider-ovirt/vendor/github.com/hashicorp/terraform/plugin.(*ResourceProviderServer).Apply(0xc0002005e0, 0xc0003fa940, 0xc0003fd860, 0x0, 0x0)
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/git/go/src/github.com/imjoey/terraform-provider-ovirt/vendor/github.com/hashicorp/terraform/plugin/resource_provider.go:530 +0x57
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: reflect.Value.call(0xc0000ccd80, 0xc0004ec1e0, 0x13, 0x186a090, 0x4, 0xc000499f18, 0x3, 0x3, 0xc0004de180, 0xc0001a2780, ...)
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/lang/go/src/reflect/value.go:447 +0x449
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: reflect.Value.Call(0xc0000ccd80, 0xc0004ec1e0, 0x13, 0xc000536f18, 0x3, 0x3, 0x0, 0xc0004e10e0, 0x0)
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/lang/go/src/reflect/value.go:308 +0xa4
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: net/rpc.(*service).call(0xc0004de980, 0xc0004ea3c0, 0xc0004f2370, 0xc0004f2380, 0xc00046d880, 0xc0004e8b60, 0x14de8c0, 0xc0003fa940, 0x16, 0x14de900, ...)
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/lang/go/src/net/rpc/server.go:384 +0x14e
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: created by net/rpc.(*Server).ServeCodec
2019-06-03T15:57:22.886-0400 [DEBUG] plugin.terraform-provider-ovirt: 	/home/jooho/dev/lang/go/src/net/rpc/server.go:481 +0x47e
2019/06/03 15:57:22 [TRACE] root: eval: *terraform.EvalWriteState
2019/06/03 15:57:22 [TRACE] root: eval: *terraform.EvalApplyProvisioners
2019/06/03 15:57:22 [TRACE] root: eval: *terraform.EvalIf
2019/06/03 15:57:22 [TRACE] root: eval: *terraform.EvalWriteState
2019/06/03 15:57:22 [TRACE] root: eval: *terraform.EvalWriteDiff
2019/06/03 15:57:22 [TRACE] root: eval: *terraform.EvalApplyPost
2019/06/03 15:57:22 [ERROR] root: eval: *terraform.EvalApplyPost, err: 1 error(s) occurred:

* ovirt_vm.my_vm_1: unexpected EOF
2019-06-03T15:57:22.889-0400 [DEBUG] plugin: plugin process exited: path=/home/jooho/.terraform.d/plugins/terraform-provider-ovirt
2019/06/03 15:57:22 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred:

* ovirt_vm.my_vm_1: unexpected EOF
2019/06/03 15:57:22 [TRACE] [walkApply] Exiting eval tree: ovirt_vm.my_vm_1
2019/06/03 15:57:22 [TRACE] dag/walk: upstream errored, not walking "output.vm_id"
2019/06/03 15:57:22 [TRACE] dag/walk: upstream errored, not walking "meta.count-boundary (count boundary fixup)"
2019/06/03 15:57:22 [TRACE] dag/walk: upstream errored, not walking "provider.ovirt (close)"
2019/06/03 15:57:22 [TRACE] dag/walk: upstream errored, not walking "root"
2019/06/03 15:57:22 [TRACE] Preserving existing state lineage "edb4a18e-44c9-918d-92bc-9f924a12ddcc"
2019/06/03 15:57:22 [TRACE] Preserving existing state lineage "edb4a18e-44c9-918d-92bc-9f924a12ddcc"
2019/06/03 15:57:22 [TRACE] Preserving existing state lineage "edb4a18e-44c9-918d-92bc-9f924a12ddcc"
2019/06/03 15:57:22 [TRACE] Preserving existing state lineage "edb4a18e-44c9-918d-92bc-9f924a12ddcc"
2019/06/03 15:57:22 [DEBUG] plugin: waiting for all plugin processes to complete...
2019-06-03T15:57:22.900-0400 [WARN ] plugin: error closing client during Kill: err="connection is shut down"



!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Terraform crashed! This is always indicative of a bug within Terraform.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Terraform[1] so that we can fix this.

When reporting bugs, please include your terraform version. That
information is available on the first line of crash.log. You can also
get it by running 'terraform --version' on the command line.

[1]: https://github.com/hashicorp/terraform/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

--- a/ovirt/resource_ovirt_vm.go
+++ b/ovirt/resource_ovirt_vm.go
@@ -501,9 +501,12 @@ func resourceOvirtVMRead(d *schema.ResourceData, meta interface{}) error {
 	// If the virtual machine is cloned from a template or another virtual machine,
 	// the template links to the Blank template, and the original_template is used to track history.
 	// Otherwise the template and original_template are the same.
-	templateCloned := vm.MustTemplate().MustId() != vm.MustOriginalTemplate().MustId() && vm.MustTemplate().MustId() == BlankTemplateID
+	originalTemplate, originalTemplateOk := vm.OriginalTemplate()
+	templateCloned := originalTemplateOk &&
+		vm.MustTemplate().MustId() != originalTemplate.MustId() &&
+		vm.MustTemplate().MustId() == BlankTemplateID
 	if templateCloned {
-		d.Set("template_id", vm.MustOriginalTemplate().MustId())
+		d.Set("template_id", originalTemplate.MustId())
 	} else {
 		d.Set("template_id", vm.MustTemplate().MustId())
 	}

